styx-cli/src/main/java/com/spotify/styx/cli/CliMain.java

```diff
@@ -380,6 +377,74 @@ private void workflowCreate() throws IOException, ExecutionException, Interrupte
     }
   }
 
+  private void workflowEnable() throws ExecutionException, InterruptedException {
+    final String component = namespace.getString(parser.workflowEnableComponentId.getDest());
+    final List<String> workflows = namespace.getList(parser.workflowEnableWorkflowId.getDest());
+    workflowStateUpdate(component, workflows, WorkflowState.patchEnabled(true))
+        .forEach(t -> {
+          if (t._2.enabled().orElse(false)) {
+            cliOutput.printMessage("Workflow " + t._1 + " in component " + component + " enabled.");
+          } else {
+            cliOutput.printMessage(
+                "Failed to enable workflow " + t._1 + " in component " + component + ".");
+          }
+        });
+  }
+
+  private void workflowDisable() throws ExecutionException, InterruptedException {
+    final String component = namespace.getString(parser.workflowDisableComponentId.getDest());
+    final List<String> workflows = namespace.getList(parser.workflowDisableWorkflowId.getDest());
+    workflowStateUpdate(component, workflows, WorkflowState.patchEnabled(false))
+        .forEach(t -> {
+          if (!t._2.enabled().orElse(false)) {
```

Comment:
this seems clunky… shouldn't we be able to trust the http status code to say whether the operation succeeded or not?

Reply:
Maybe we can. Our server side code doesn't really check whether the change is successful, and in case of concurrent modification http code can be lying. Maybe too paranoid.

Reply:
I tend to prefer relying on the http status code as the standard unique way to assess correctness of an API call and move all the verification steps on the server side (before determining the status code to return).
I'd be happy with just removing this check from the CLI for now and address the verification step for `WorkflowState` cahnges on the server side in a future PR, if really needed.

Reply:
Cool let’s do that then. Two votes now. 😀